robot/impl/local_robot_test.go

```diff
@@ -2303,33 +2295,41 @@ func TestCheckMaxInstanceSkipRemote(t *testing.T) {
 	ctx := context.Background()
 	logger := logging.NewTestLogger(t)
 
-	r0 := setupLocalRobot(t, ctx, &config.Config{}, logger)
+	remoteConfig := setupLocalRobot(t, ctx, &config.Config{
```

Comment:
Can you explain the change here?

Reply:
Basically, the naming was incorrect since previously r0 was actually the remote, so I change the names and explicitly added a data manager to both the remote and non-remote. Then checked to see that two datamanagers were successfully made since having a DM on the remote + non-remote should violate the 1 DM rule we usually have.

Reply:
Ahhh I see, that's great and makes sense with the below checks - thank you for explaining.